packages/email-components/src/components/Rating/README.md

```diff
@@ -0,0 +1,32 @@
+# Rating
+
+The `mjc-rating` and `mjc-rating-item` are components for user rating and setting the status of jobs offered in the e-mail template.
```

Comment:
This sentence seems unclear to me. Can you please re-text it in a way to better understand user rating and setting the status?

Reply:
Is it now better?